### DISCUSSION TITLE
I am experiencing following no matching manifest for windows/amd64 10.0.14393 in the manifest list entries on windows server 2016

### DISCUSSION THREAD

**Discussion by EmilSahin88** [docker-library/wordpress#800] [General]
Hi guys

Can someone help me?

I am experiencing following no matching manifest for windows/amd64 10.0.14393 in the manifest list entries on windows server 2016.

This is the docker-compose.yml file I am using:

`version: '3.9'

services:
  db:
    image: mariadb:10.6.11
    restart: always
    environment:
      MARIADB_ROOT_PASSWORD: XKN0Z34LCR4gHmKzJ9GAuY7gMoXt8LMd
      MARIADB_DATABASE: wp_nunagis_hub_db
      MARIADB_USER: wp_nunagis_adm
      MARIADB_PASSWORD: URmCTMjVObLoQSJkRANOG5z7cqNgkFmO
    volumes:
      - ./mariadb:/var/lib/mysql
    networks:
      - wp_nunagis_hub_net

  phpmyadmin:
    depends_on:
      - db
    image: phpmyadmin:5.2
    restart: always
    ports:
      - 9080:80
    environment:
      PMA_HOST: db
      MYSQL_ROOT_PASSWORD: XKN0Z34LCR4gHmKzJ9GAuY7gMoXt8LMd
    networks:
      - wp_nunagis_hub_net

  wordpress:
    depends_on:
      - db
    image: wordpress:6.1.1
    restart: always
    ports:
      - 8080:80
    environment:
      WORDPRESS_DB_HOST: db
      WORDPRESS_DB_USER: wp_nunagis_adm
      WORDPRESS_DB_PASSWORD: URmCTMjVObLoQSJkRANOG5z7cqNgkFmO
      WORDPRESS_DB_NAME: wp_nunagis_hub_db
    volumes:
      - ./wordpress:/var/www/html
    networks:
      - wp_nunagis_hub_net

volumes:
  wordpress:
  db:
networks:
  wp_nunagis_hub_net:`

**Comment by tianon** [comment]
Unfortunately, we do not maintain any native Windows container images -- you'll probably want to switch to Linux containers instead.

**Reply by EmilSahin88** [reply]
Thnak you. I will try that
